src/main/java/sg/ncl/MainController.java

```diff
@@ -1280,6 +1301,7 @@ public String removeExperiment(@PathVariable String expId, Model model, HttpSess
 
         HttpEntity<String> request = new HttpEntity<String>("parameters", headers);
         ResponseEntity responseEntity = restTemplate.exchange(properties.getDeleteExperiment(expId), HttpMethod.POST, request, String.class);
+        System.out.println(responseEntity.getBody().toString());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)